Description: remove -lssl from LIBS as it is not needed
 no symbols from libssl are used.
Origin: vendor
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=80368
Forwarded: yes
Author: Damyan Ivanov <dmn@debian.org>
Last-Update: 2011-08-05

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -6,6 +6,6 @@ WriteMakefile(
     'VERSION_FROM'	=> 'lib/Crypt/OpenSSL/DSA.pm', # finds $VERSION
     'PREREQ_PM'		=> {}, # e.g., Module::Name => 1.1
     'AUTHOR'            => 'T.J. Mather',
-    'LIBS'              => ['-lssl -lcrypto'],
+    'LIBS'              => ['-lcrypto'],
     'DEFINE'            => '-DPERL5',   # perl-5.8/gcc-3.2 needs this
 );
